tests/rptest/tests/debug_bundle_test.py

```diff
@@ -68,6 +70,15 @@ def bundle_status():
 
         wait_until(bundle_status, timeout_sec=60, backoff_sec=1)
 
+        res = admin.get_debug_bundle(node=node)
+        assert res.status_code == requests.codes.ok
+        local_bundle_filename = "copy-bundle.zip"
+        with open(local_bundle_filename, "wb") as local_bundle:
+            local_bundle.write(res.content)
+
+        # The file was downloaded to orchestrator node
+        assert os.path.isfile(local_bundle_filename)
+
```

Comment:
My manual test was more strict.
1. Initiate the debug bundle
```bash
curl -s -X POST "http://localhost:9644/v1/debug/bundle/actions/create"
```

2. Wait for debug bundle to be ready
```bash
curl -s -X GET "http://localhost:9644/v1/debug/bundle/files"

debug-bundle.zip (finished)
```

3. Download bundle with curl and redirect to a new filename
```bash
curl -s -X GET "http://localhost:9644/v1/debug/bundle" > copy-bundle.zip
```

4. Verify the copy is the same as the original (this is on my local so it's easy to do this):
```bash
diff -q /mnt/nvme1n1p1/local-cluster/debug-bundle.zip copy-bundle.zip ; echo $?

0
```

Maybe we should include a checksum or something so users can verify the bundle isn't corrupted?

Reply:
Using `wget` in step (3) does not work, only part of the file is downloaed. I'll need to figure that out later. The whole file is